### PR TITLE
Add Ganesh's Pittsford File to Master

### LINKED ISSUE #6: Create a file for Ganesh

### PR DESCRIPTION
**Pull request by gmasilamani**
closes #6

### CODE REVIEW

File: gmasilamani-pittsford.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Pittsford, NY
+
+#### Places to go
+- NiagaraFalls
+#### Places to eat
```
Comment (esaezsa) [inline gmasilamani-pittsford.md:8]: What if I get hungry? :hamburger:

Comment (gmasilamani) [reply gmasilamani-pittsford.md:8]: go to wegmans

**Comment by gmasilamani** [comment]
@esaezsa is it good to go?

**Comment by gmasilamani** [comment]
@YashSV is it good to go?

**Comment by YashSV** [comment]
:+1:

**Pull request merged**